js/build.js

```diff
@@ -114,16 +103,21 @@ $('[data-login-ds-id]').each(function() {
       dataSourceId: data_source_id,
       where: where
     })
-      .catch(function (error) {
-        return Promise.reject(error);
-      });
+    .catch(function (error) {
```

Comment:
You chained the `.then()` in the same line as the function. You shoudl either move the `.then()` to a new line, or move the `.catch()` to the previous line -- for consistency.